modules/oc-mirror-imageset-config-params.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-imageset-config-params_{context}"]
+= Image set mirroring configuration parameters
+
+// TODO intro
+
+// TODO: Updated all of the "mappings" to "String" like elsewhere in our doc - okay?
+// TODO: removed "Kind" row - okay with that?
+
+.`ImageSetConfiguration` parameters
+[cols="2,2,1a",options="header"]
+|===
+|Parameter
+|Description
+|Values
+
+|`apiVersion`
+|The API version for the `ImageSetConfiguration` content.
+|String. For example: `mirror.openshift.io/v1alpha1`.
+
+// TODO: Changed example (lowered the "v") - confirm
+
+|`archiveSize`
+|The maximum size, in GiB, of each archive file within the image set.
+|String. For example: `4.6`
+
+|`mirror`
+|The configuration of the image set.
+|Object
+
+|`mirror.additionalImages`
+|The additional images configuration of the image set.
+|Array of objects. For example:
+
+[source,yaml]
+----
+additionalImages:
```

Comment:
@afflom @jpower432 The Google doc had a lowercase "i" in the example here ("additionalimages"). I capitalized it to "additionalImages". Let me know if that is correct.